molecule.yml

```diff
@@ -31,6 +31,10 @@ ansible:
 
 # configuration options for the internal call to vagrant
 vagrant:
+  raw_config_args:
```

Comment:
I'm not sure we should include `landrush` again. In my opinion, it doesn't work _that_ well and these lines force people to have it installed (if don't, molecule won't test the role), and we have to add the tld. I suggest using `vagrant-hostsupdater`

Reply:
done